src/algebra/category/Module/monoidal.lean

```diff
@@ -3,16 +3,16 @@ Copyright (c) 2020 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kevin Buzzard, Scott Morrison
 -/
-import category_theory.monoidal.category
+import category_theory.monoidal.braided
 import algebra.category.Module.basic
 import linear_algebra.tensor_product
 
 /-!
-# The monoidal category structure on R-modules
+# The symmetric monoidal category structure on R-modules
 
 Mostly this uses existing machinery in `linear_algebra.tensor_product`.
 We just need to provide a few small missing pieces to build the
-`monoidal_category` instance.
+`monoidal_category` instance and then the `symmetric_monoidal_category` instance.
```

Comment:
```suggestion
`monoidal_category` instance and then the `braided_monoidal_category` instance.
```
Maybe also upstairs?

Reply:
(Actually it is `symmetric_category`, I'd just named it incorrectly. It's also a bit confusing because the symmetry field is provided by `tidy`.)